<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSoeHistoryWgt</class>
 <widget class="QWidget" name="CSoeHistoryWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSoeHistoryWgt</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>条目信息</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_6">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>事件类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>条目数</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>进度</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>保护动作事件</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>开入变位事件</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>异常事件</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>运行事件</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>录波事件</string>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>调试信息</string>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string>0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>动作</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>SOE时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>事件类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>动作状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>故障侧值</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>变位</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <column>
          <property name="text">
           <string>SOE时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>遥信类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>变位名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>变位类型</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>异常</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_3">
         <column>
          <property name="text">
           <string>SOE时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>异常信息</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>运行</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_4">
         <column>
          <property name="text">
           <string>SOE时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>运行信息</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>录波</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget_5">
         <column>
          <property name="text">
           <string>SOE时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>录波文件名</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
